pkgs/by-name/co/cortex-tools/package.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule (finalAttrs: {
+  pname = "cortex-tools";
+  version = "0.11.3";
+
+  src = fetchFromGitHub {
+    owner = "grafana";
+    repo = finalAttrs.pname;
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-+GWUC+lnCn5Nw2WytSvW/UsIMmMelCCsnKdBCHuue24=";
+  };
+
+  vendorHash = null;
+
+  subPackages = [
+    "cmd/benchtool"
+    "cmd/cortextool"
+    "cmd/e2ealerting"
+    "cmd/logtool"
+  ];
+
+  meta = {
+    changelog = "https://github.com/grafana/cortex-tools/releases/tag/${finalAttrs.src.tag}";
+    description = "Tools used for interacting with Cortex, a Prometheus-compatible server";
+    longDescription = ''
+      Tools used for interacting with Cortex, a horizontally scalable, highly available, multi-tenant, long term Prometheus server:
+
+      - benchtool: A powerful YAML driven tool for benchmarking Cortex write and query API.
+      - cortextool: Interacts with user-facing Cortex APIs and backend storage components.
+      - logtool: Tool which parses Cortex query-frontend logs and formats them for easy analysis.
+      - e2ealerting: Tool that helps measure how long an alert takes from scrape of sample to Alertmanager notification delivery.
+    '';
+    homepage = "https://github.com/grafana/cortex-tools";
```

Comment:
is there a sane `mainProgram` to set for this tool? `cortextool`? Or are they all equally "main"?